File: library/src/main/java/com/devbrackets/android/androidmarkup/parser/SpanType.java
package com.devbrackets.android.androidmarkup.parser;

/**
 * Holds the values associated with specific types of spans
 * for the {@link MarkupParser}.  If you need to implement custom
 * spans, use values after {@link #MAX_SPAN_TYPE}
 */
public interface SpanType {
    int BOLD = 1;
    int ITALIC = 2;
    int ORDERED_LIST = 3;
    int UNORDERED_LIST = 4;
    int MAX_SPAN_TYPE = UNORDERED_LIST;
}

File: library/src/main/kotlin/com/devbrackets/android/androidmarkup/parser/core/SpanType.kt
package com.devbrackets.android.androidmarkup.parser.core

/**
 * Holds the values associated with specific types of spans
 * for the [MarkupParser].  If you need to implement custom
 * spans, use values after [.MAX_SPAN_TYPE]
 */
object SpanType {
    const val UNKNOWN = 0
    const val TEXT = 1
    const val BOLD = 2
    const val ITALIC = 3
    const val ORDERED_LIST = 4
    const val UNORDERED_LIST = 5
    const val MAX_SPAN_TYPE = UNORDERED_LIST
}
